### PR TITLE
Redirect to /user/organizations after login

### PR DESCRIPTION
**Pull request by maukoquiroga**
Partial fix for #44

### CODE REVIEW

File: lib/transport_web/controllers/session_controller.ex [hunk 1]
```diff
@@ -15,19 +15,21 @@ defmodule TransportWeb.SessionController do
...
 
   def create(conn, %{"code" => code}) do
-    client = get_token!(code)
-    user   = get_user!(client)
-
-    conn
-    |> put_session(:current_user, user)
-    |> put_session(:access_token, client.token.access_token)
-    |> redirect(to: "/search_organizations")
+    with client <- get_token!(code),
```
Comment (l-vincent-l) [inline lib/transport_web/controllers/session_controller.ex]: You need to do:
```elixir
with {:ok, client} <- get_token(code),
    {:ok, user} <- get_client(code) do
…
else

File: test/transport_web/controllers/page_controller_test.exs [hunk 2]
```diff
@@ -1,18 +1,26 @@
 defmodule TransportWeb.PageControllerTest do
   use TransportWeb.ConnCase
 
+  doctest TransportWeb.PageController
+
   test "GET /", %{conn: conn} do
-    conn = get conn, "/"
-    assert html_response(conn, 200) =~ "Rendre disponible, valoriser et améliorer les données transports"
+    with path <- conn |> page_path(:index),
+         conn <- conn |> get(path) do
```
Comment (l-vincent-l) [inline test/transport_web/controllers/page_controller_test.exs]: you don't need a `with` statement here.

File: test/transport_web/controllers/page_controller_test.exs [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+      assert html_response(conn, 200) =~ "disponible, valoriser et améliorer"
+    end
   end
 
   test "GET /search_organizations", %{conn: conn} do
-    conn = get conn, "/search_organizations"
-    assert html_response(conn, 200) =~ "<organization-search></organization-search>"
+    with path <- conn |> page_path(:search_organizations),
+         conn <- conn |> get(path) do
```
Comment (l-vincent-l) [inline test/transport_web/controllers/page_controller_test.exs]: Same here

File: test/transport_web/controllers/session_controller_test.exs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+defmodule TransportWeb.SessionControllerTest do
+  use TransportWeb.ConnCase
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  doctest TransportWeb.SessionController
+
+  test "GET /login/callback", %{conn: conn} do
+    use_cassette "session/create-2" do
+      with path <- conn |> session_path(:create, %{"code" => "secret"}),
+           conn <- conn |> get(path) do
```
Comment (l-vincent-l) [inline test/transport_web/controllers/session_controller_test.exs]: same here

File: lib/transport_web/controllers/session_controller.ex [hunk 5]
```diff
@@ -15,19 +15,21 @@ defmodule TransportWeb.SessionController do
...
+         user <- get_user!(client) do
+      conn
+      |> put_session(:current_user, user)
+      |> put_session(:access_token, client.token.access_token)
+      |> redirect(to: user_path(conn, :organizations))
+      |> halt()
```
Comment (l-vincent-l) [inline lib/transport_web/controllers/session_controller.ex]: You're not in a plug, so you don't need a `halt`

Comment (maukoquiroga) [reply lib/transport_web/controllers/session_controller.ex]: No.

**Pull request merged**